src/openms/include/OpenMS/FORMAT/DATAACCESS/SwathFileConsumer.h

```diff
@@ -240,7 +256,8 @@ namespace OpenMS
         {
           // We group by the precursor mz (center of the window) since this
           // should be present in all SWATH scans.
-          if (std::fabs(center - swath_map_boundaries_[i].center) < 1e-6)
+          // also specify ion mobility, if ion mobility not present will just be -1
+          if ((std::fabs(center - swath_map_boundaries_[i].center) < 1e-6) && (std::fabs(lowerIm - swath_map_boundaries_[i].imLower) < 1e-6) && (std::fabs(upperIm - swath_map_boundaries_[i].imUpper < 1e-6)))
```

Comment:
the bracket is wrong in the last expression. should close before the comparison

Reply:
can you add a test for this path?

Reply:
Thanks for the catch. I fixed the bracket placement (https://github.com/OpenMS/OpenMS/pull/5911/commits/e4b7a5121360cd5d63bde4d3ab03fce3a701d340). What were you thinking in terms of a test here, because I think the window placement is already properly tested in the openswath workflow test?